server/lib/questionnaire-validator.js

```diff
@@ -4,9 +4,10 @@ import Questionnaire from '../models/questionnaire'
 import validate from '../../common/validators'
 
 export default function getValidator(identifier) {
-  if (process.env.NODE_ENV === 'test') return () => true
 
   return async function validator(fields) {
+    if (fields.length === 0) return true
```

Comment:
If there's a required field in the questionnaire and you submit no fields this will pass when it should fail won't it?